recipes/open62541/all/conanfile.py

```diff
@@ -170,6 +177,11 @@ def configure(self):
         if self.options.web_socket:
             self.options["libwebsockets"].with_ssl = self.options.encryption
 
+        if self.options.nodeset_loader:
```

Comment:
As this option is removed in Windows, accessing it should be done thru a safe getter method
```suggestion
        if self.options.get_safe("nodeset_loader"):
```

Note that this is the case for every instance of option access in the PR, they should also be udpated!

Reply:
Thank you for pointing this out. I have applied your suggestion in [commit 18ab99c4](https://github.com/conan-io/conan-center-index/pull/27372/commits/18ab99c497f2f477ab163f76b4cc53e9dfcba1dd). Additionally, I successfully built it locally both with the option removed and with the option enabled.